tests/integration/startup/startup_int_test.go

```diff
@@ -365,7 +365,32 @@ var _ = Describe("startup tests", Ordered, func() {
 			Expect(testutil.K3sCleanup(-1, "")).To(Succeed())
 		})
 	})
-
+	When("a server with a --deny-psa-label is created", func() {
+		It("is created with no arguments", func() {
+			var err error
+			startupServerArgs = []string{
+				"--deny-psa-label",
+				"--kube-apiserver-arg",
+				"enable-admission-plugins=DenyPSALabel",
+			}
+			startupServer, err = testutil.K3sStartServer(startupServerArgs...)
+			Expect(err).ToNot(HaveOccurred())
+		})
+		It("has the default pods deployed", func() {
+			Eventually(func() error {
+				return testutil.K3sDefaultDeployments()
+			}, "120s", "5s").Should(Succeed())
+		})
+		It("change label of namespace", func() {
+			res, err := testutil.K3sCmd("kubectl label --dry-run=server --overwrite ns --all pod-security.kubernetes.io/enforce=baseline")
+			Expect(err).To(HaveOccurred())
+			Expect(res).To(ContainSubstring("denying use of PSA label on namespace"))
```

Comment:
I think that this test will fail as-is, since this is not the string that is output any longer.
```suggestion
			Expect(res).To(ContainSubstring("Use of label with pod-security.kubernetes.io/ prefix on Namespace is denied by admission control"))
```